examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -129,7 +129,7 @@
       },
       "outputs": [],
       "source": [
-        "os.environ['OPENAI_API_KEY'] = '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'"
```

Comment:
Also, definitely delete and cycle this if you haven't already!

Reply:
Thank You. I have deleted it.